src/ucp/core/ucp_ep.c

```diff
@@ -1964,7 +1970,9 @@ static ucs_status_t ucp_ep_config_calc_params(ucp_worker_h worker,
             }
         }
 
-        bw = ucp_tl_iface_bandwidth(context, &iface_attr->bandwidth);
+        bw = ucp_tl_iface_bandwidth(context, &iface_attr->bandwidth) /
```

Comment:
maybe use `ucp_tl_iface_bandwidth_ratio` or create some other common function which would take `multi_path_ratio` into account

Reply:
per offline discussion we cannot use ucp_tl_iface_bandwidth_ratio function in this flow.
@brminich can we keep the PR as is then?